plugins/fileservices/fileservices.cpp

```diff
@@ -1096,6 +1098,94 @@ FILESERVICES_API char * FILESERVICES_CALL fsfSprayXml_v4(ICodeContext *ctx, cons
 
 //----------------------------------------------------------------------------------
 
+FILESERVICES_API char * FILESERVICES_CALL implementSprayJson(ICodeContext *ctx, const char * sourceIP, const char * sourcePath, int sourceMaxRecordSize, const char *sourceRowPath, const char *sourceEncoding, const char * destinationGroup, const char * destinationLogicalName, int timeOut, const char * espServerIpPort, int maxConnections, bool overwrite, bool replicate, bool compress, bool failIfNoSourceFile, int expireDays, const char * dfuServerQueue, bool noSplit, const char * username, const char * userPw)
+{
```

Comment:
This is very similar (a lot of the lines are identical) to implementSprayXml,
it would be good to put common code into a helper for both to use.

In fact I think the other sprays implement* function share a lot in common too.